layer/filestore.go

```diff
@@ -328,10 +328,12 @@ func (fms *fileMetadataStore) getOrphan() ([]roLayer, error) {
 					contentBytes, err := ioutil.ReadFile(chainFile)
 					if err != nil {
 						logrus.WithError(err).WithField("digest", dgst).Error("cannot get cache ID")
```

Comment:
Assuming this could fail if `chainFile` is gone (perhaps a partial cleanup happened before?); wondering if we should just silently ignore that case. The `driver.Cleanup()` will clean up the directory anyway, correct?
```suggestion
					if !os.IsNotExist(err) {
						logrus.WithError(err).WithField("digest", dgst).Error("failed to read cache ID")
					}
```